themes/bira.zsh-theme

```diff
@@ -20,8 +22,9 @@ fi
 local git_branch='$(git_prompt_info)%{$reset_color%}'
 
 PROMPT="╭─${user_host} ${current_dir} ${rvm_ruby} ${git_branch}
-╰─%B$%b "
-RPS1="${return_code}"
+╰─%B${user_symbol}%b "
```

Comment:
It's shorter using `%(!.#.$)` (see [prompt expansion](http://zsh.sourceforge.net/Doc/Release/Prompt-Expansion.html#Shell-state)). 

The previous `user_host` stuff could be similarly abbreviated with a `%(!.$fg[red].$fg[green])`, but let's leave that for another time.